### PR TITLE
[Fix Post Blog] - Demistifying Open Source

### LINKED ISSUE #5446: [Post Blog] - Demistifying Open Source
Contributors and community members are encouraged to post on https://layer5.io/blog. A new blog post fitting the following description is requested to be written by a community member:

**Desired Title:** 
Demistifying Open Source

**Topic:** 
A Personal Journey and Insights into Getting Started

**Core points to convey in the post:** 
1. Getting Started: Choosing a project to contribute to
1. What skills do I need to get started?
1. My Open Source Journey: From Curiosity to Contribution
1. How to Get Started with Meshery - Making your first contributions
1. Reflections on Personal and Professional Growth

---
<h3>Contributor Resources</h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by sandramsc**
**Description**

This PR fixes #5446 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
build failing...

File: src/collections/blog/2024/Demystifying Open Source/index.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+title: Demystifying Open Source
+subtitle: A Personal Journey and Insights into Getting Started
+date: 2024-03-10 08:00:00 -0530
+author: Sandra Ashipala
+thumbnail: ./banner-hero.png
+darkthumbnail: ./banner.png
```
Comment (vishalvivekm) [inline src/collections/blog/2024/Demystifying Open Source/index.mdx:7]:
```suggestion
thumbnail: ./banner-hero.png
darkthumbnail: ./banner.png
```

These PNGs are missing

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
Also, please consider renaming the directory from `Demystifying Open Source` to `demystifying-open-source`

**Comment by sandramsc** [comment]
@vishalvivekm Regarding points above, this PR is in **draft** mode and not ready for review yet, thanks.

**Comment by l5io** [comment]
🚀 Preview for commit e0e645a9f64a66025c6860043a97d3bf7fab9f4c at: https://65f1aa2d3e64391fa5b71b35--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit df36b224642ba9a8c8171f88d42826e18cae7931 at: https://65f1becc5b8293312897488f--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit bf68d636318677f69b3988e95c0e058bad02a149 at: https://65f1c802db39543babe16f50--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9198225032acbd0da030d8d3fd8ca5162e820695 at: https://65f2d8477ef5a9b59a42117e--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 289e9f4f3386085d61f94e92d848c0a7f083e2d8 at: https://65f3a6ea0171f35e1d92aeec--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@sandramsc Let's discuss this on websites call. Add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#), if you would :)

File: package-lock.json [hunk 2]
```diff
@@ -7652,25 +7652,6 @@
         "lodash.uniq": "^4.5.0"
       }
     },
-    "node_modules/caniuse-lite": {
```
Comment (iArchitSharma) [inline package-lock.json:7655]: hey @sandramsc , can you please revert this file??

**Review by iArchitSharma — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 2 above)
Comment (sandramsc) [reply package-lock.json:7655]: @iArchitSharma just so I understand the request, you mean to remove it yes?

**Comment by sudhanshutech** [comment]
yes @sandramsc remove any changes from package file until there is no change in packages

**Comment by l5io** [comment]
🚀 Preview for commit 7c2dce4a7c56f2ab6e53aeb230b3789b0b69e6cc at: https://666f22797ab324ccebde871f--layer5.netlify.app

**Review by sudhanshutech — CHANGES_REQUESTED** [review]
remove changes from package.json please

**Comment by sudhanshutech** [comment]
merge conflicts...

**Comment by sudhanshutech** [comment]
is this in still draft @sandramsc ?

**Pull request closed without merging**